docs/Developers/Writing_Good_Commits.rst

```diff
@@ -150,6 +150,8 @@ Example:
    If a specific issue may lead to an unintended behaviour from the user
    or from the program it should be considered a bug, and should be
    addresed with ``Fixes``.
+-  If an issue is labelled with ``bug`` you should always use ``Fixes``.
+-  For all other issues use ``Closes``.
```

Comment:
Dont you think this is what is written in the line above. IMO you should explain the reason for using Fixes.

Reply:
The reason for Fixes/Closes is already explained well enough in the whole documentation. This just mentions the specifics , like to always use Fixes for bug labelled issues @newbazz

Reply:
Ok well then sorry ;).